docs/en/sdk-manual/user-identifier.md

```diff
@@ -0,0 +1,68 @@
+
+{{ solution_name }} solution mainly contains three types of IDs, namely:
+
+- User ID
+- Device ID
+- User Pseudo ID
+
+Next, we will introduce these three IDs respectively, and you will learn in detail how we use User ID and User Pseudo ID to correlate user behavior.
+
+## User ID
+
+User ID is usually a unique identifier that describes the user in your business database, which is relatively more accurate and unique.
+
+* When the user is not registered or logged in, the value of User ID is empty.
+* The SDK provides the `ClickstreamAnalytics.setUserId("your user id")` method to set the User ID. When logging out, set `null/nil` to clear the User ID.
+* The User ID is stored in the `user_id` field in the user table.
+
+## Device ID
+
+We identify user devices by Device ID.
+
+* The Device ID will be automatically generated when the app is launched for the first time after integrating the SDK.
+* The Device ID may not be the unique identifier of the device. Usually the Device ID may be regenerated after the user uninstalls the app or clears the cache on the web page.
+* The Device ID is stored in the `device_id_list` field in the user table.
+
+The following table will introduce how the SDK on each end generates the Device ID.
+
+| SDK Types    | Generate Rules                                                                                                                                                            | Storage Location                                | Is Unique                                                                                                                                                    |
+|--------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Android SDK  | By default, AndroidId is used as the Device ID. If the AndroidId cannot be obtained, a random UUID is used instead                                                        | Stored in SharedPreference key-value pair file  | Usually the AndroidId will not change when the app is uninstalled. <br>If the UUID is obtained, the Device ID will change after the user uninstalls the app  |
+| Swift SDK    | If the user has been authorized to obtain IDFA, use IDFA as the Device ID. Otherwise, use IDFV as the Device ID by default. If IDFV cannot be obtained, use a random UUID | UserDefault key-value pair file                 | Usually the IDFA does not change when the app is uninstalled. <br>When using IDFV or UUID, the Device ID will change after the user uninstalls the app       |
```

Comment:
-> If your app has been authorized to obtain IDFA, use IDFA as the Device ID. Otherwise, use IDFV as the Device ID. If IDFV cannot be obtained, use a random UUID.

Reply:
-> Usually, the IDFA does not change even if the app is uninstalled and reinstalled.
When using IDFV or UUID, the Device ID will change after the user uninstalls and reinstalls the app.

Reply:
fixed.